service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/ManageDocumentsUploadedEventHandler.java

```diff
@@ -0,0 +1,341 @@
+                            furtherEvidenceNotificationService.sendNotification(event.getCaseData(), recipients,
+                                event.getInitiatedBy().getFullName(), newDocumentNames);
+                        }
+                    }
+                }
+            });
+    }
+
+    @Async
+    @EventListener
+    public void sendDocumentsByPost(final ManageDocumentsUploadedEvent event) {
+        final CaseData caseData = event.getCaseData();
+        DocumentUploaderType userType = event.getUploadedUserType();
+
+        if (userType == SOLICITOR) {
+            List<DocumentReference> nonConfidentialPdfDocumentsToBeSent =
+                consolidateMapByConfiguration(event, (docType) -> ConfidentialLevel.NON_CONFIDENTIAL)
+                    .entrySet().stream()
+                    .filter(entry -> !isHearingDocument(entry.getKey()))
+                    .map(Map.Entry::getValue)
+                    .flatMap(List::stream)
+                    .map(Element::getValue)
+                    .map(NotifyDocumentUploaded::getDocument)
+                    .filter(documentReference -> hasExtension(documentReference.getFilename(), PDF))
+                    .collect(toList());
+
+            if (!nonConfidentialPdfDocumentsToBeSent.isEmpty()) {
+                Set<Recipient> allRecipients = new LinkedHashSet<>(sendDocumentService.getStandardRecipients(caseData));
+                sendDocumentService.sendDocuments(caseData, nonConfidentialPdfDocumentsToBeSent,
+                    new ArrayList<>(allRecipients));
+            }
+        }
+    }
+
+    @Retryable(value = EmailFailedSendException.class)
+    @Async
+    @EventListener
+    public void sendDocumentsToCafcass(final ManageDocumentsUploadedEvent event) {
+        final CaseData caseData = event.getCaseData();
+
+        if (CafcassHelper.isNotifyingCafcassEngland(caseData, cafcassLookupConfiguration)) {
+            Map<DocumentType, List<Element<NotifyDocumentUploaded>>> documentsToBeSent =
+                consolidateMapByConfiguration(event, DocumentUploadedNotificationConfiguration::getSendToCafcassEngland)
+                    .entrySet().stream()
+                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+
+            if (!documentsToBeSent.isEmpty()) {
+                Map<CafcassRequestEmailContentProvider, List<DocumentType>> emailTemplateMap =
+                    documentsToBeSent.keySet().stream()
+                        .collect(groupingBy(docType ->
+                                docType.getNotificationConfiguration().getCafcassRequestEmailContentProvider(),
+                                toList()));
+
+                emailTemplateMap.forEach((cafcassEmailContentProvider, documentTypeList) -> {
+                    String documentTypes = documentTypeList.stream()
+                        .map(docType ->
+                            String.join(" ", LIST, docType.getDescription().replaceAll("└─ ", "")))
```

Comment:
perhaps don't need `LIST`? The name is a bit ambiguous.

Reply:
refactored it to `BULLET_POINT`